app/src/main/java/a3/com/convo/activities/ProfileActivity.java

```diff
@@ -11,10 +11,6 @@
 import a3.com.convo.fragments.ProfileDetailsFragment;
 
 public class ProfileActivity extends AppCompatActivity {
-
-    private AddInfoFragment addInfoFrag;
-    private ProfileDetailsFragment profileDetailsFrag;
-    private Fragment profileFrag;
     private FragmentTransaction ft;
```

Comment:
Make this local variable?

Reply:
This is no longer in my code, I must have selected older code on accident during merge.

Reply:
Can you rebase and push